### PR TITLE
Responsive Main Menu #107  And Game Play Instructions #109 

### PR DESCRIPTION
**Pull request by kamishettysreeja25**
@niccokunzmann 
I have made the following changes
1- I have Changed the main_menu.css file Which takes care of re sizing the buttons
2-I have added the image instructions.png file which contains the instructions so that i can reduce the css file of the howtoplay.html
If There is any thing more to do please suggest me :)

### CODE REVIEW

File: howtoplay.html [hunk 1]
```diff
@@ -5,9 +5,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="icon" type="image/ico" href="images/favicon.ico" />
     </head>
+    <body bgcolor="#ffcc7e">
     <body>
         <p>
-        <h1>This Page contains the game rules </h1>
+        <img src="images/instructions.png" alt="Mountain View" style="width:35%;height:100%;position:absolute;left:35%;">
```
Comment (niccokunzmann) [inline howtoplay.html]: Can you move the style into the css file?

**Comment by kamishettysreeja25** [comment]
Would U like me to add a css file for the howtoplay.html ? I thought It would need a extra file

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 This would be good. But before you do it: Please use Inkscape to create the picture and save it as SVG.

**Comment by kamishettysreeja25** [comment]
For Now I have Changed the howtoplay.html and Added howtoplay.css file. I am working on the svg image for the controls  and if the how to play  information for the game play is fine i would like to continue with them

File: style/main_menu.css [hunk 2]
```diff
@@ -25,19 +56,20 @@ li a:hover {
...
+    left: 30%;
+    height: 110%;
+    padding-top: 0px;
 }
 body {
-    background-color: black;
+    background-color: #ffcc7e;
     margin: 0px 400px 0px 400px;
     padding: 0;
     text-align: center;
```
Comment (niccokunzmann) [inline style/main_menu.css]: You can create improvement pull-requests any time. This could be three:

1. main menu
2. credits
3. howtoplay

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
I have changed the png file to svg please review it :) 
If anything more is required please tell me

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 The design is fine. I would like to merge it.

Please always consider that people may use smaller screens and test your site for that. Mobile phones have a smaller resolution with more height that width.
capture
In this case the border overlaps the image.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
I have understood the problem I have made the position of texts  and background absolute . But till I am facing the same problem there is a overlapping .Do You have any better way of doing this ? :)

**Comment by niccokunzmann** [comment]
The page would just contain this image at 100% width. That is all.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann  
Now the border doesn't ovarlap with the image . Hope it fixes the problem :)

**Comment by kamishettysreeja25** [comment]
@niccokunzmann  
I have separated the css and html part.

File: style/howtoplay.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+body{ 
+	background-color: #ffcc7e;
```
Comment (niccokunzmann) [inline style/howtoplay.css]: Do we need the following four lines and what are the for?

	width:35%;
	height:100%%;
	position:absolute;
	left:35%;

**Comment by kamishettysreeja25** [comment]
This is for the image to be positioned   towards right of the left margin by 35% and setting the width and height of the image 

width:35%;
height:100%;
position:absolute;
left:35%;

this configuration is same for main menu also  , so that it looks similar to the main menu .

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 
If you make the window smaller, the image gets cut off.
I think we can circumvent this by using

    max-width: 40%;
    max-height: 100%;

What do you think?

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
the image is still getting cut off ater replacing the widht to max-width and height to max-heigth
screenshot from 2016-03-06 01 28 23

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 Do you think it is ok if it is cut off this way?

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
It will fine, as user may not resize the window to so small size and this works fine in the app .
so i think this will not be any problem :)

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 ok :)

**Comment by kamishettysreeja25** [comment]
so we can now close the issue ,  closes #109

**Pull request merged**